_includes/layouts/example.html

```diff
@@ -8,7 +8,7 @@ <h1>{{ title }}</h1>
   <time datetime="{{ page.date | htmlDateString }}">{{ page.date | readableDate }}</time>
   {%- if author %}
   <span class="author">
-    by&nbsp;<span class="author">{{ author }}</span>
+    by&nbsp;<span class="author"><a href="/contributors/{{ author }}">{{ author }}</a></span>
```

Comment:
Please use the `url` filter for propper URL generation, like here for example:
https://github.com/tbreuss/mithril-by-examples/blob/main/_includes/layouts/example.html#L31

Reply:
@tbreuss yeah i understand that, but in don't know how to add contributors url using that.

Reply:
@tbreuss i don't understand how to get contributors/author url using what you say.

Reply:
Please see https://www.11ty.dev/docs/filters/url/

Reply:
@tbreuss Thanks, i have used slug and filter now, please review this, it seems fine to me now.

Reply:
Looks good, will test it and then merge, thanks

Reply:
@tbreuss cool, but do it as soon as possible because hacktoberfest is gonna end today and i'm left with one PR so i don't want this PR to not get considered, Welcome btw!

Reply:
Just tested, works great. Thanks for the PR.